tests/test_geometry.py

```diff
@@ -0,0 +1,220 @@
+import numpy
+
+import collada
+from collada.util import unittest
+from collada.xmlutil import etree
+
+fromstring = etree.fromstring
+tostring = etree.tostring
+
+
+class TestGeometry(unittest.TestCase):
```

Comment:
Is it possible to avoid rewriting this file in the diff, maybe by moving some of the reformatting to a separate PR?